packages/itwin/reports-config-widget/src/widget/components/ActionPanel.tsx

```diff
@@ -35,16 +34,15 @@ const ActionPanel = ({
       >
         {actionLabel}
       </Button>
-      <Button
-        styleType="default"
-        type="button"
+      {onCancel && <Button
+        styleType='default'
+        type='button'
+        id='cancel'
```

Comment:
Why do you need the id here? Such a generic id is almost a guarantee to conflict with some other id. Ids should be unique throughout the whole application.

Reply:
id removed. Was not needed.